lib/ExtUtils/MM_Unix.pm

```diff
@@ -2896,6 +2896,7 @@ Returns true, if the argument is likely to be a command.
 
 sub maybe_command {
     my($self,$file) = @_;
+    $file = '' if (!defined $file or !length $file);
```

Comment:
This could be simply `$file = '' unless defined $file;`.

The only value that's `!length` is `''`, so in that case the assignment is redundant.

Reply:
Or even just `return unless defined $file`, since zero-length-name files can't exist.